src/meltano/core/logging/utils.py

```diff
@@ -121,6 +121,13 @@ def default_config(log_level: str) -> dict:
                 "level": logging.INFO,
                 "propagate": False,
             },
+            # Azure HTTP logs at info level are too noisy; see
+            # https://github.com/meltano/meltano/issues/7723
+            "azure.core.pipeline.policies.http_logging_policy": {
+                "handlers": ["console"],
+                "level": logging.WARNING,
+                "propagate": False,
+            },
```

Comment:
I think we can also achieve the desired behavior if we omit `handlers` and enable `propagate`

```suggestion
                "level": logging.WARNING,
                "propagate": True,
            },
```